src/components/Modal/Modal.jsx

```diff
@@ -55,6 +56,7 @@ const Modal = (props) => {
         <div className={styles.Modal__image} style={style} />
         <div className={styles.Modal__information}>
           <h2>{attraction.name}</h2>
+          <Favorite favId={attraction.placeId} />
```

Comment:
Let's wrap this in a feature toggle.

Pass `Feature` to `Modal` as a prop `Feature={Feature}`. Destructure off of props and add to validation.

Then wrap in `Feature.Toggle`
```
<Feature.Toggle flag="heart-fav">
  <Favorite favId={attraction.placeId} />
</Feature.Toggle>
```